sys/kern/src/arch/rv64/task.rs

```diff
@@ -9,26 +9,17 @@ use crate::umem::USlice;
 
 use core::arch::asm;
```

Comment:
Github action is complaining of this unused import as well.

Reply:
That's unexpected - we're using asm!(...) inside macros in this file.

Reply:
Probably it was because of the typo in the usage of `jump_to_task` for M-mode.